core/src/main/java/de/jplag/GreedyStringTiling.java

```diff
@@ -30,6 +31,13 @@ public class GreedyStringTiling {
     private final Map<Submission, int[]> cachedTokenValueLists = new IdentityHashMap<>();
     private final Map<Submission, SubsequenceHashLookupTable> cachedHashLookupTables = new IdentityHashMap<>();
 
+    private static final String ERROR_INDEX_OUT_OF_BOUNDS = """
```

Comment:
Elaborate what index and length is printed.
Add information what two submissions are being compared (names)

Reply:
I would second that.